src/msg/async/AsyncConnection.cc

```diff
@@ -977,9 +977,8 @@ ssize_t AsyncConnection::_process_connection()
                                 << " not " << peer_addr
                                 << " - presumably this is the same node!" << dendl;
           } else {
-            ldout(async_msgr->cct, 0) << __func__ << " connect claims to be "
-                                << paddr << " not " << peer_addr << " - wrong node!" << dendl;
-            goto fail;
+            ldout(async_msgr->cct, 20) << __func__ << " connect claims to be "
```

Comment:
I think its a little noisy too -- happy to remove it. Maybe @liewegas can weigh in here.